### ISSUE TITLE
UseHttpLogging not working .NET 6 RC 1

### ISSUE THREAD

**Issue by no-stack-dub-sack** [dotnet/aspnetcore#37577]
### Describe the bug
I don't believe this can actually be the case, as I would likely be far from the first person to report it, however, `app.UseHttpLogging()` seems to have no effect on logging for me regardless of how I configure the middleware with `builder.Services.AddHttpLogging(o => ...)`. Normally I might take this to SO, but thought this might be appropriate given .NET 6 has not been officially released yet.

I can see the app's startup logs in the debug console, so can confirm that logging in general is working:

```
Microsoft.Hosting.Lifetime: Information: Now listening on: https://localhost:5001
info: Microsoft.Hosting.Lifetime[14]
      Now listening on: https://localhost:5001
info: Microsoft.Hosting.Lifetime[14]
      Now listening on: http://localhost:5000
Microsoft.Hosting.Lifetime: Information: Now listening on: http://localhost:5000
```

### To Reproduce
[Here ](https://github.com/no-stack-dub-sack/dotnet-issue-repro) is a link to a minimal repro with which I'm experiencing this issue. My actual project is only slightly more complex with db contexts, auth, and a couple of other middlewares and endpoints, none of which seem to be the reason, since its still not working when I boil it down to this minimal example.

I've checked out the docs [here](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/http-logging/?view=aspnetcore-6.0) but have not seen any reason why this shouldn't just work out of the box when calling `app.UseHttpLogging()`.

Thank you.

### Further technical details
- ASP.NET Core version: .NET 6 RC1
- The IDE (VS / VS Code/ VS4Mac) you're running on, and its version: VS Code 1.60.2
- Include the output of `dotnet --info`:

<details>
  <summary>dotnet --info Output</summary>

```
.NET SDK (reflecting any global.json):
 Version:   6.0.100-rc.1.21463.6
 Commit:    e627d556a1

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.18363
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   C:\Program Files\dotnet\sdk\6.0.100-rc.1.21463.6\

Host (useful for support):
  Version: 6.0.0-rc.1.21451.13
  Commit:  d7619cd4b1

.NET SDKs installed:
  2.1.104 [C:\Program Files\dotnet\sdk]
  2.2.109 [C:\Program Files\dotnet\sdk]
  3.1.201 [C:\Program Files\dotnet\sdk]
  5.0.400 [C:\Program Files\dotnet\sdk]
  6.0.100-rc.1.21463.6 [C:\Program Files\dotnet\sdk]

.NET runtimes installed:
  Microsoft.AspNetCore.All 2.1.29 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.All]
  Microsoft.AspNetCore.All 2.2.7 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.All]
  Microsoft.AspNetCore.App 2.1.29 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 2.2.7 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 3.1.3 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 3.1.18 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 5.0.9 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 6.0.0-rc.1.21452.15 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 2.0.6 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.1.29 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.2.7 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 3.1.3 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 3.1.13 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 3.1.18 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 5.0.9 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 6.0.0-rc.1.21451.13 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.WindowsDesktop.App 3.1.3 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 3.1.13 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 3.1.18 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 5.0.9 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 6.0.0-rc.1.21451.3 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]

To install additional .NET runtimes or SDKs:
  https://aka.ms/dotnet-download
```

</details>

**Comment by BrennanConroy** [comment]
You need to have information level logging for HttpLogging to show up, right now you're set at warning level
https://github.com/no-stack-dub-sack/dotnet-issue-repro/blob/ac42ab81e5e634024641404d56b3f45e385187f7/Repro/appsettings.Development.json#L5

**Comment by no-stack-dub-sack** [comment]
Ah, that was definitely it. I thought I checked that file and verified that the level was set to "Information" but I must have only looked at the default level, not for Microsoft. I didn't think to double check this because other informational logs in the Microsoft namespace were coming through, but of course that's because they're specifically set that way. Thanks, I knew I had to be missing something obvious.
